Formula/xcode-kotlin.rb

```diff
@@ -0,0 +1,28 @@
+class XcodeKotlin < Formula
+  desc "Kotlin Native Xcode Plugin"
+  homepage "https://github.com/touchlab/xcode-kotlin"
+  url "https://github.com/touchlab/xcode-kotlin.git",
+    tag:      "1.2.0",
+    revision: "3af7e5d921c3756a3adc83ad504ac0db2a5643e8"
+  license "Apache-2.0"
+  head "https://github.com/touchlab/xcode-kotlin.git", branch: "main"
+  depends_on "gradle" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on :macos
```

Comment:
Let's keep some space here:
```suggestion
  head "https://github.com/touchlab/xcode-kotlin.git", branch: "main"

  depends_on "gradle" => :build
  depends_on xcode: ["12.5", :build]
  depends_on :macos
```